### DISCUSSION TITLE
error while pushing image using buildah push

### DISCUSSION THREAD

**Discussion by rakeshgm** [containers/buildah#4741] [Q&A]
I am trying to build an operator for different architectures. After I run the below command  
`buildah bud --manifest ${IMG} --arch s390x`  the manifest file is correctly created, but when I run `buildah push ${IMG}` I get the following error.
Referencing from [here](https://sdk.operatorframework.io/docs/advanced-topics/multi-arch/#manifest-lists) 
```
Error: pushing image "docker.io/repo/image:tag" to "docker://docker.io/repo/image:tag": no image found in image index for architecture amd64, variant "", OS linux: image not known
```
I am not sure why amd64 is being selected when I am building for `s390x` 
any help with what could be wrong here? 

```
buildah --version
buildah version 1.29.1 (image-spec 1.0.2-dev, runtime-spec 1.0.2-dev)
```

**Comment by abdelghanimeliani** [comment]
hey @rakeshgm im not sure , but i think you should specify the destination path (registry/username/imagename:tag) and the creds to push
